staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1589,6 +1593,12 @@ message ContainerEventResponse {
 
     // ID of the sandbox container
     PodSandboxMetadata pod_sandbox_metadata = 4;
+
+    // Sandbox statuses
+    PodSandboxStatus pod_sandbox_status = 5;
```

Comment:
sorry I missed this in previous review. `PodSandboxMetadata` is already present in `PodSandboxStatus` so no need to include it in `ContainerEventResponse`.

https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto#L515

Reply:
Good point, I will update the code. Thanks.

Reply:
Updated.